src/content/hinting.ts

```diff
@@ -905,11 +905,8 @@ function filterHintsVimperator(fstr, reflow = false) {
     // Start with all hints
     let active = modeState.hints
 
-    // If we're reflowing, the names may be wrong at this point, so apply the original names.
-    if (reflow) rename(active)
```

Comment:
@cmcaine do you know why you thought the names might be wrong? We now hint JS-event hints separately so it is no longer (this) trivial to re-apply the original names.

In my limited testing it works fine without this line.